config/locales/en.yml

```diff
@@ -133,26 +119,24 @@ en:
           `- [download option 1](/link)`
 
           `- [download option 2](/link)`
-    reorder_steps:
-      revewer_2i:
-        Step order
-      shared:
+    reorder:
+      header: Order of steps
+      order:
         Steps and tasks should be listed in the order in which users need to complete them.
-      origin:
-        Use the up/down buttons on the right to reorder steps, or click and hold on a step to reorder using drag and drop.
-        Step numbers will be automatically updated on save.
     sidebar_settings:
-      shared:
-        Showing or hiding the sidebar:
+      header: Sidebar settings
```

Comment:
It looks like this is nested under "secondary_content_links". Should this be under sidebar_settings on line 67?

Reply:
Sorry. Now that I'm looking at the whole file, I can see how you've separated the contextual and full page guidance headings.